map_model/src/edits/mod.rs

```diff
@@ -606,6 +621,88 @@ fn recalculate_intersection_polygon(
     affected
 }
 
+fn fix_buildings(map: &mut Map, input: Vec<BuildingID>) {
+    // TODO Copying from make/buildings.rs
+    let mut center_per_bldg: BTreeMap<BuildingID, HashablePt2D> = BTreeMap::new();
+    let mut query: HashSet<HashablePt2D> = HashSet::new();
+    for id in input {
+        let center = map.get_b(id).polygon.center().to_hashable();
+        center_per_bldg.insert(id, center);
+        query.insert(center);
+    }
+
+    // equiv_pos could be a little closer, so use two buffers
```

Comment:
equiv_pos could be a little closer to what?
What are the two buffers? I see just `sidewalk_buffer`

Reply:
This looks like a stale comment copied around. Removing